src/helpers/_grid.scss

```diff
@@ -23,6 +23,7 @@
 
 @mixin create-grid-row($gutter) {
     display: flex;
+    flex: 1 1 0;
     flex-wrap: wrap;
     margin-right: $gutter / -2;
     margin-left: $gutter / -2;
```

Comment:
IE wordt over een tijdje niet meer ondersteund. Is er een voorbeeld waar deze fix voor nodig is? Ik heb het zelf nog niet eerder nodig gehad. Misschien is het mogelijk om dit in het project op te lossen. Ik kijk graag even met je mee.

Reply:
In ie11 en andere versies van windows edge staat de waardes voor flex op 0 0 auto als default. Hierdoor reageert flexbox op ie heel anders dan in chrome waarbij flex default op 1 staat. 

Door hard 1 aan te geven werkt flexbox op ie 11 hetzelfde als op chrome. 

IE11 wordt misschien niet ondersteund functioneel moet het wel werken. Vandaar deze oplossing.

Reply:
Dat begrijp ik, maar hebben jullie hier een praktijkvoorbeeld van? Ik ben gewoon niewsgierig wat er precies moet worden opgelost.

Reply:
Ja ik wil een afbeelding erin plaatsen maar kan het niet vinden  hoe:(

Reply:
Als je een comment schrijft en die textarea actief/open houdt in github, dan kun je jouw filemanager op de pc openen en de afbeelding naar deze textarea slepen, dan upload github het automatisch :slightly_smiling_face: 

Werkt trouwens ook met .gif-bestanden :smile: 

![cat-at-laptop](https://user-images.githubusercontent.com/57255942/97959875-8e132100-1db0-11eb-8704-d40efec7f6bc.jpeg)

Reply:
Mij viel vooral deze passage op: `The problem is that this solution will probably mess up firefox` 😬 Dus zeker nog even checken op FF of het goed gaat, want het zou zonde zijn als we een uitfaserend fossiel ondersteunen maar een moderne browser buggen :)